.docker/php/docker-entrypoint.sh

```diff
@@ -16,24 +16,6 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
         composer install --prefer-dist --no-progress --no-interaction || true
     fi
 
-    if grep -q DATABASE_URL= .env; then
-        echo "Waiting for database to be ready..."
-        ATTEMPTS_LEFT_TO_REACH_DATABASE=30
-        until [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ] || DATABASE_ERROR=$(php bin/console dbal:run-sql -q "SELECT 1" 2>&1); do
-            sleep 1
-            ATTEMPTS_LEFT_TO_REACH_DATABASE=$((ATTEMPTS_LEFT_TO_REACH_DATABASE - 1))
-            echo "Still waiting for database to be ready... Or maybe the database is not reachable. $ATTEMPTS_LEFT_TO_REACH_DATABASE attempts left."
-        done
-
-        if [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ]; then
-            echo "The database is not up or not reachable:"
-            echo "$DATABASE_ERROR"
-            exit 1
-        else
-            echo "The database is now ready and reachable"
-        fi
-    fi
-
```

Comment:
This might cause an issue when setting up the app outside of dev stack for the first time but that should be pretty rare occurrence so lets see.